### DISCUSSION TITLE
How delete file uploading after run test?

### DISCUSSION THREAD

**Discussion by ssheduardo** [spatie/laravel-medialibrary#2199] [Q&A]
I created a test for create a Banner with a image using medialibrary, the test passed but, how delete the image uploaded the file after run test?

**Test passed**
![image](https://user-images.githubusercontent.com/1160138/100074300-dcab5c80-2e3e-11eb-9fe3-937d25519b62.png)

**Code test**
![image](https://user-images.githubusercontent.com/1160138/100074321-e0d77a00-2e3e-11eb-9724-fa0f805a5a8a.png)

**Public**
![image](https://user-images.githubusercontent.com/1160138/100074343-e6cd5b00-2e3e-11eb-8ab2-836bd4fbfc45.png)

**Controller**
![image](https://user-images.githubusercontent.com/1160138/100074440-095f7400-2e3f-11eb-9a1e-3831b30e70bf.png)

Once the image has been automatically uploaded and passed the test, how can the file be automatically deleted once the test has been executed?

**Comment by ignacio-dev** [comment]
I know this is a very old question, but in case somebody needs an answer, you can just run ```Storage::fake('public');``` again at the end of your test, and it will delete the files :)

So:

```
public function test_something()
{
    Storage::fake('public');
    // Your test code ...
    Storage::fake('public');
}
```
